Sources/PointFreeRouter/Routes.swift

```diff
@@ -151,12 +156,17 @@ let routers: [Router<Route>] = [
   .case { .blog(.show($0)) }
     <¢> get %> "blog" %> "posts" %> pathParam(.blogPostIdOrString) <% end,
 
-  .case(Route.episode)
-    <¢> get %> "episodes" %> pathParam(.episodeIdOrString) <% end,
-
-  .case(.episodes)
+  .case(.episode(.index))
     <¢> get %> "episodes" <% end,
 
+  parenthesize(.case { Route.episode(Route.EpisodeRoute.progress(param: $0, percent: $1)) })
```

Comment:
Worth simplifying to?
```suggestion
  parenthesize(.case { .episode(.progress(param: $0, percent: $1)) })
```
Or does type checking start to suffer?

Reply:
I think I only had that while trying to get things to compile.